test/functional/autocmd/cursorhold_spec.lua

```diff
@@ -0,0 +1,31 @@
+local helpers = require('test.functional.helpers')(after_each)
+
+local clear = helpers.clear
+local eq = helpers.eq
+local eval = helpers.eval
+local feed = helpers.feed
+local retry = helpers.retry
+local source = helpers.source
+local sleep = helpers.sleep
+
+describe('CursorHoldI', function()
+  before_each(clear)
+
+  -- NOTE: since this test uses RPC it is not necessary to trigger the initial
+  --       issue (#3757) via timer's or RPC callbacks in the first place.
+  it('is triggered after input', function()
```

Comment:
i would put this test in https://github.com/neovim/neovim/blob/31f879983f5effb94a8d62c3073576511131bf6e/test/functional/api/vim_spec.lua#L613

Reply:
Should I move it?

Reply:
👍